docs/core/testing/unit-testing-platform-config.md

```diff
@@ -38,3 +40,7 @@ Environment variables can be used to supply some runtime configuration informati
 
 > [!NOTE]
 > Environment variables take precedence over configuration settings in the *testconfig.json* file.
+
+### `TESTINGPLATFORM_UI_LANGUAGE` environment variable
```

Comment:
Is this the right doc page for documenting this? @Evangelink

Reply:
@Evangelink This is the docs for `DOTNET_CLI_UI_LANGUAGE`
https://github.com/dotnet/docs/blob/3208829cb687c603ab97762f8cfad9aa12dd3c4c/docs/core/tools/dotnet-environment-variables.md?plain=1#L335-L337

Reply:
We can copy the first part:

```
Sets the language of the CLI UI using a locale value such as `en-us`. The supported values are the same as for Visual Studio. For more information, see the section on changing the installer language in the [Visual Studio installation documentation](/visualstudio/install/install-visual-studio).
```

In the `dotnet test` integration doc we could link to this part of the CLI doc.